### ISSUE TITLE
add battery value (few) for non EV car ?

### ISSUE THREAD

**Issue by MZorzy** [msp1974/homeassistant-jlrincontrol#41]
hi,
looking on my log here https://github.com/msp1974/homeassistant-jlrincontrol/issues/11#issuecomment-630030269 or stefan here https://github.com/msp1974/homeassistant-jlrincontrol/issues/13#issuecomment-615894875 , we have few data about Battery on petrol/diesel car :+1: 
 
'TU_STATUS_PRIMARY_VOLT': '3.9000000000000004',
'TU_STATUS_PRIMARY_CHARGE_PERCENT': '100'
'BATTERY_STATUS': 'BATTERY_1_1'

may will be add it?

**Comment by msp1974** [comment]
Whats the use case for this? Anything starting TU is for the telematics unit and it manages its battery level by going to sleep if the car is not used for some time. As such, theres not really any action you need to take from the info provided.

There is also a car battery level, but again, on modern cars you dont really need to worry about it unless the car tells you to. I also dont know what the different status messages mean. If anyone can help translate them and feel they are useful, happy to add battery sensor for this.

I have on the list to look at some way to capture vehicle alerts and maybe issue as HA notifications, but havent had time to look at yet.

**Comment by MZorzy** [comment]
Yes there is not any action we can do with it.
but on sensor.rrs_hst_info have stuff like this:

`
Registration ....
Model  Year2019
Make Range Rover
Model Sport
Body 5 Door Estate
Colour Silver
Doors 5
Seats 5
Engine 3.0
Transmission Automatic
Fuel Petrol
Weight 2318
`

so i only ask to see on HA all data that JLS server send to us.

**Comment by msp1974** [comment]
@MZorzy Your issue got me thinking about people who have their own specific requirements that it may not be right to include in the integration as it is a specific use case etc.

I have created a new All Info sensor (released in v2.1 today) that supplies all the info from attributes, status and position in one sensor.  Whilst not in itself useful (although maybe interesting), it will allow creation of custom sensors for anyone who wishes to have something different.

In the recipes document, I have provided an example of how to create such a custom sensor.

Let me know if you think this is good or a rubbish answer to your requirements.
